catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/AimidbAttribute.java

```diff
@@ -0,0 +1,117 @@
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+
+public enum AimidbAttribute implements NitfAttribute<Tre> {
+    ACQUISITION_DATE_AND_TIME("acquisitionDateAndTime",
+            "ACQUISITION_DATE",
+            tre -> TreUtility.getTreValue(tre, "ACQUISITION_DATE"),
+            BasicTypes.DATE_TYPE),
+    MISSION_NUMBER("missionNumber", "MISSION_NO", tre -> TreUtility.getTreValue(tre, "MISSION_NO")),
+    MISSION_IDENTIFICATION("missionIdentification",
+            "MISSION_IDENTIFICATION",
+            tre -> TreUtility.getTreValue(tre, "MISSION_IDENTIFICATION")),
+    FLIGHT_NUMBER("flightNumber", "FLIGHT_NO", tre -> TreUtility.getTreValue(tre, "FLIGHT_NO")),
+    IMAGE_OPERATION_NUMBER("imageOperationNumber",
+            "OP_NUM",
+            tre -> TreUtility.getTreValue(tre, "OP_NUM"),
+            BasicTypes.INTEGER_TYPE),
+    CURRENT_SEGMENT_ID("currentSegementId",
+            "CURRENT_SEGMENT",
+            tre -> TreUtility.getTreValue(tre, "CURRENT_SEGMENT")),
+    REPROCESS_NUMBER("reprocessNumber",
+            "REPRO_NUM",
+            tre -> TreUtility.getTreValue(tre, "REPRO_NUM"),
+            BasicTypes.INTEGER_TYPE),
+    REPLAY("replay", "REPLAY", tre -> TreUtility.getTreValue(tre, "REPLAY")),
+    START_TILE_COLUMN_NUMBER("startTileColumnNumber",
+            "START_TILE_COLUMN",
+            tre -> TreUtility.getTreValue(tre, "START_TILE_COLUMN"),
+            BasicTypes.INTEGER_TYPE),
+    START_TILE_ROW_NUMBER("startTileRowNumber",
+            "START_TILE_ROW",
+            tre -> TreUtility.getTreValue(tre, "START_TILE_ROW"),
+            BasicTypes.INTEGER_TYPE),
+    ENDING_SEGMENT("endingSegment",
+            "END_SEGMENT",
+            tre -> TreUtility.getTreValue(tre, "END_SEGMENT")),
+    END_TILE_COLUMN_NUMBER("endTileColumnNumner",
+            "END_TILE_COLUMN",
+            tre -> TreUtility.getTreValue(tre, "END_TILE_COLUMN"),
+            BasicTypes.INTEGER_TYPE),
+    END_TILE_ROW_NUMBER("endTileRowNumber",
+            "END_TILE_COLUMN",
+            tre -> TreUtility.getTreValue(tre, "END_TILE_ROW"),
+            BasicTypes.INTEGER_TYPE),
+    COUNTRY("country", "COUNTRY", tre -> TreUtility.getTreValue(tre, "COUNTRY")),
+    LOCATION("location", "LOCATION", tre -> TreUtility.getTreValue(tre, "LOCATION"));
+
+    private static final String ATTRIBUTE_NAME_PREFIX = "nitf.aimidb.";
+
+    private String shortName;
+
+    private String longName;
+
+    private Function<Tre, Serializable> accessorFunction;
+
+    private AttributeDescriptor attributeDescriptor;
+
+    AimidbAttribute(String longName, String shortName,
+            Function<Tre, Serializable> accessorFunction) {
+        this(longName, shortName, accessorFunction, BasicTypes.STRING_TYPE);
+    }
+
+    AimidbAttribute(final String lName, final String sName,
+            final Function<Tre, Serializable> function, AttributeType attributeType) {
```

Comment:
Modify this constructor to take either the `MetacardType` or `AttributeDescriptor` (similar to what was done above in `AcftbAttribute` rather than the `AttributeType`.